ts/client/src/accounts/group.ts

```diff
@@ -472,11 +470,23 @@ export class Group {
       uiPrice = this.toUiPrice(price, baseDecimals);
       lastUpdatedSlot = stubOracle.lastUpdateSlot.toNumber();
       provider = OracleProvider.Stub;
+      deviation = stubOracle.deviation;
     } else if (isPythOracle(ai)) {
       const priceData = parsePriceData(ai.data);
       uiPrice = priceData.previousPrice;
       price = this.toNativePrice(uiPrice, baseDecimals);
       lastUpdatedSlot = parseInt(priceData.lastSlot.toString());
+      deviation = priceData.confidence
+        ? I80F48.fromNumber(
+            priceData.confidence *
+              Math.pow(
+                10,
+                priceData.exponent +
```

Comment:
did you verify this one? priceData.previousPrice seems to already have the exponent factored in, so it's odd that this needs it to be applied manually